source/git-util.js

```diff
@@ -116,7 +116,10 @@ const gitVersion = async () => {
 };
 
 exports.verifyRecentGitVersion = async () => {
-	if (semver.lt(await gitVersion(), '2.11.0')) {
+	const installedVersion = await gitVersion();
+	const minVersion = '2.11.0';
+
+	if (!(version.isVersionGreaterThan(minVersion, installedVersion) || version.isVersionEqualTo(minVersion, installedVersion))) {
```

Comment:
This is really verbose. I think we should have `version.isVersionGreaterThanOrEqualTo()` method.

Reply:
Also, from taking another look at the codebase, it seems like having `version.isGreaterThanOrEqualTo()` and `version.isLowerThanOrEqualTo()` would be enough to cover all current cases of version comparison.